xbmc/cores/AudioEngine/Sinks/AESinkPULSE.h

```diff
@@ -48,7 +50,9 @@ class CAESinkPULSE : public IAESink
   bool IsInitialized();
   void UpdateInternalVolume(const pa_cvolume* nVol);
   pa_stream* GetInternalStream();
+  pa_threaded_mainloop* GetInternalMainLoop();
   CCriticalSection m_sec;
+  std::atomic<unsigned int> m_requestedBytes;
```

Comment:
unsigned int is a bad idea :)

Reply:
https://freedesktop.org/software/pulseaudio/doxygen/stream_8h.html#abe284dcb2ffbee97145fb10c328d5731 <- the callback parameter is a size_t

but you are right, when I add periodSize_bytes it could easily overflow and end up with a very large value. Let's take int